proc/groupby.go

```diff
@@ -125,10 +134,12 @@ func NewGroupByAggregator(c *Context, params GroupByParams) *GroupByAggregator {
 		limit = defaultGroupByLimit
 	}
 	return &GroupByAggregator{
-		keys:            params.keys,
-		dt:              c.Resolver,
-		reducerDefs:     params.reducers,
-		typeCols:        make(map[string][]zeek.TypedEncoding),
+		keys:        params.keys,
+		dt:          c.Resolver,
+		reducerDefs: params.reducers,
+		// keysMap maps an input descriptor to a descriptor
```

Comment:
We don't ever actually use the set of groupby keys as a descriptor do we?
Is this just for expedience since the Mapper api works with descriptors?

Reply:
I think its a reasonable approach

Reply:
Yeah, I wasn't objecting, was just trying to understand.  I think it would be useful to clarify this in a comment.